### DISCUSSION TITLE
horizontal space - can I make this smaller?

### DISCUSSION THREAD

**Discussion by jniggemann** [TriliumNext/Notes#549] [General]
Latest TriliumNext on Windows, but it's the same in the browser. I think we're wasting a bit of screen real estate here - can we make this gap smaller?
![grafik](https://github.com/user-attachments/assets/66f7f975-6a7e-4608-a99f-e0d2e8dc741e)

**Comment by eliandoran** [comment]
@jniggemann, the size of the area of text is called the "content width". It is possible to adjust it from Options -> Appearance -> Content Width.

![image](https://github.com/user-attachments/assets/4baa7008-3e03-4dc6-b492-26d20cc21166)

Increasing the width to a higher value should reduce the tag you are seeing. It's even possible to set it to a very high value to have it always have this behaviour.

If you are interested in doing this only for a single note, use the `#fullContentWidth` attribute.

**Comment by jniggemann** [comment]
Oh, got it - TY! 👍
